components/brave_vpn/browser/brave_vpn_service_unittest.cc

```diff
@@ -529,7 +529,7 @@ class BraveVPNServiceTest : public testing::Test {
 TEST_F(BraveVPNServiceTest, ResponseSanitizingTest) {
   // Give invalid json data as a server response and check sanitized(empty
   // string) result is returned.
-  SetInterceptorResponse("{'a':'b',}");
```

Comment:
the string was a actually valid JSON (taking into account the brave patches to data_decoder).